Formula/termshark.rb

```diff
@@ -0,0 +1,24 @@
+class Termshark < Formula
+  desc "Terminal UI for tshark, inspired by Wireshark"
+  homepage "https://termshark.io"
+  url "https://github.com/gcla/termshark/archive/v1.0.0.tar.gz"
+  sha256 "669bba0e8dd7df54ade6321a5c7d2ec20563ffd777f7b3b0394a11f88da64698"
+
+  depends_on "go@1.11" => :build
```

Comment:
Can't this build with the latest go? I'm a bit hesitant to add a formula with an already outdated dependency.

Reply:
Hi - yes, it can build with the latest go. 1.11 is the minimum really - go with modules support. Looking at depends_on more closely, I think I can express that 1.11 is a minimum version required - I'll try that.

Reply:
@gcla: Just `depends_on "go" => :build` should do the trick. No need to specify a minimum version. Versions monotonically increase in homebrew 😉 (unless someone messes up 🤦‍♂)

```suggestion
  depends_on "go" => :build
```

Reply:
Thanks, the latest push makes this change.